Week-2/Advanced CSS/Recipe with flexbox/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>binary eats - Oatmeal Coconut Pantry Cookies</title>
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+
+  <body>
+    <header>
+      <div class="logo">binary eats</div>
+      <div>
+        <a href="">sign-in</a>
+        <a href="">search</a>
```

Comment:
<a href-"#">sign</a>

Adding a # in the href attr of anchor tag makes the link go nowhere.

Reply:
The template was already provided for this exercise, and CSS had to be added to meet the requirements.